src/storage/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2018"
 name = "fluvio-storage"
-version = "0.5.0"
+version = "0.4.0"
```

Comment:
no need to change this version

Reply:
I just think that since the last published version on crates.io is 0.3.0 that it makes no sense to jump to 0.5.0, I don't see a reason why not to

Reply:
k